app/src/main/res/drawable/continent_south_america.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="378.4156"
+    android:viewportHeight="610.44"
+    android:width="378.41562dp"
+    android:height="610.44dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.